pkg/accounting/rollup/rollup.go

```diff
@@ -102,12 +106,12 @@ func (r *Service) RollupStorage(ctx context.Context, lastRollup time.Time, rollu
 	//loop through tallies and build Rollup
 	for _, tallyRow := range tallies {
 		node := tallyRow.NodeID
-		if tallyRow.CreatedAt.After(latestTally) {
-			latestTally = tallyRow.CreatedAt
+		interval := tallyRow.IntervalEndTime.UTC()
```

Comment:
what sets `IntervalEndTime` ?

Reply:
`IntervalEndTime` is set to `time.Now()` right _before_ the return of `CalculateAtRestData`. `CreatedAt` is set when `time.Now()` right _after_ the return of `CalculateAtRestData` only nanoseconds later.
This change wasn't totally necessary since we'll be refactoring anyway to use the `storagenode_storage_tally` table which only has `interval_start`

Reply:
any chance we can name `interval` something more descriptive since its hard to understand whats going on here. Like `tallyEndTime` or something. Also a comment would probably be helpful too. All these timestamps are confusing, at least you got rid of createdAt, one less timestamp! yay

Reply:
Update here, I'm leaving the CreatedAt timestamp in storagenode_storage_tally https://github.com/storj/storj/pull/1769

Reply:
But if we decide to no longer use the createdAt timestamp let me know